src/inmanta/config.py

```diff
@@ -207,6 +208,17 @@ def is_time(value: str) -> int:
     return int(value)
 
 
+def is_time_or_cron(value: str) -> Union[int, str]:
+    """Time, the number of seconds represented as an integer value"""
+    if isinstance(value, int):
```

Comment:
I don't quite understand the issue it looks to me that Config.set("config", "agent-repair-splay-time", "2") is already being set in the test ?
I tried to reproduce but I couldn't, it looks to me value is always a string here? I think we can get rid of the `isinstance` check and update the docstring to reflect that `value` can also be a cron expression.

Reply:
I made a mistake, I was talking about agent-repair-interval.
you need to remove the line so that this part of the code is used:
```
agent_repair_interval = Option(
    "config",
    "agent-repair-interval",
    600,
    "The number of seconds between two repair runs (full deploy) of the agent. "
    + "Set this to 0 to disable the scheduled repair runs.",
    is_time,
)
```
and here 600 isn't a string

Reply:
Yeah, the typing is all wrong for the config/option framework... Since it's not trivial to clean up I would just follow the approach of the `is_bool` validator: accept `Union[int, str]`. Your implementation already matches that, your type signature does not.